### PR TITLE
Tomee 2330 backport 7 0

### PR DESCRIPTION
**Pull request by otaviojava**

### CODE REVIEW

**Comment by brunobat** [comment]
I'm running a full build with Java 7

File: container/openejb-core/src/test/java/org/apache/openejb/core/webservices/JPACMDIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,325 @@
+package org.apache.openejb.core.webservices;
```
Comment (brunobat) [inline container/openejb-core/src/test/java/org/apache/openejb/core/webservices/JPACMDIntegrationTest.java]: This file is missing a header

File: arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/java/org/apache/openejb/arquillian/tests/cmp/MyRemoteObject.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package org.apache.openejb.arquillian.tests.cmp;
```
Comment (brunobat) [inline arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/java/org/apache/openejb/arquillian/tests/cmp/MyRemoteObject.java]: Missing the Header

File: arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/java/org/apache/openejb/arquillian/tests/cmp/MyRemoteHome.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package org.apache.openejb.arquillian.tests.cmp;
```
Comment (brunobat) [inline arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/java/org/apache/openejb/arquillian/tests/cmp/MyRemoteHome.java]: Missing the Header

File: arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/java/org/apache/openejb/arquillian/tests/cmp/MyLocalObject.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package org.apache.openejb.arquillian.tests.cmp;
```
Comment (brunobat) [inline arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/java/org/apache/openejb/arquillian/tests/cmp/MyLocalObject.java]: Missing the Header

File: arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/java/org/apache/openejb/arquillian/tests/cmp/MyLocalHome.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
+package org.apache.openejb.arquillian.tests.cmp;
```
Comment (brunobat) [inline arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/java/org/apache/openejb/arquillian/tests/cmp/MyLocalHome.java]: Missing the Header

File: arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/java/org/apache/openejb/arquillian/tests/cmp/MyCmpBean.java [hunk 6]
```diff
@@ -0,0 +1,53 @@
+package org.apache.openejb.arquillian.tests.cmp;
```
Comment (brunobat) [inline arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/java/org/apache/openejb/arquillian/tests/cmp/MyCmpBean.java]: Missing the Header

File: arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/java/org/apache/openejb/arquillian/tests/cmp/CmpServlet.java [hunk 7]
```diff
@@ -0,0 +1,30 @@
+package org.apache.openejb.arquillian.tests.cmp;
```
Comment (brunobat) [inline arquillian/arquillian-tomee-tests/arquillian-tomee-webprofile-tests/src/test/java/org/apache/openejb/arquillian/tests/cmp/CmpServlet.java]: Missing the Header

**Comment by otaviojava** [comment]
@brunobat fixed

**Comment by brunobat** [comment]
The full build has some **unrelated** tests failing:
MaxSizePoolEndpointHandlerTest.shouldSendMessage:109 expected:<30> but was:<1>
AutoConnectionTrackerTest.test:207 expected:<1> but was:<0>
[ERROR]   TomEECliIT.testTomEECliWithJar:79
[ERROR]   TomEECliIT.testTomEECliWithJarCallingAnotherMain:174
MessageDrivenBeanInterceptorInvocationTest>Arquillian.run:164->testMessageDrivenBeanMethodIntercepted:92 expected [true] but found [false]

The build hanged on me after a while. Running again.

**Pull request merged**